lib/passenger.rb

```diff
@@ -20,5 +20,20 @@ def get_drivers
     def add_trip(trip)
       @trips << trip
     end
-  end
-end
+
+    def total_trips_cost()
+      total_cost = @trips.inject(0) do |total, trip|
+        total + trip.cost
+      end
```

Comment:
Attempting to run this method on a passenger with an unfinished trip yields an error!